gapis/api/data_group.go

```diff
@@ -79,10 +79,16 @@ func CreateEnumDataValue(typeName string, value fmt.Stringer) *DataValue {
 }
 
 func CreatePoDDataValue(ctx context.Context, s *GlobalState, typeName string, val interface{}) *DataValue {
+	var podVal *pod.Value
+	if handle, ok := val.(Handle); ok {
```

Comment:
Could we do this check inside `pod.NewValue`? I think this would be a bit cleaner and less error-prone.